Index: lib/vendor/propel-generator/classes/propel/engine/database/model/AppData.php
===================================================================
--- lib/vendor/propel-generator/classes/propel/engine/database/model/AppData.php	(revision 3150)
+++ lib/vendor/propel-generator/classes/propel/engine/database/model/AppData.php	(working copy)
@@ -95,7 +95,7 @@
      */
     public function getShortName()
     {
-        return str_replace("-schema", "", $name);
+        return str_replace("-schema", "", $this->name);
     }
 
     /**
Index: lib/vendor/propel-generator/classes/propel/engine/database/model/Table.php
===================================================================
--- lib/vendor/propel-generator/classes/propel/engine/database/model/Table.php	(revision 3150)
+++ lib/vendor/propel-generator/classes/propel/engine/database/model/Table.php	(working copy)
@@ -1015,7 +1015,7 @@
 
         if ($this->abstractValue) {
             $result .= " abstract=\""
-                  . ($abstractValue ? "true" : "false")
+                  . ($this->abstractValue ? "true" : "false")
                   . '"';
         }
 
Index: lib/vendor/propel-generator/classes/propel/phing/PropelSQLExec.php
===================================================================
--- lib/vendor/propel-generator/classes/propel/phing/PropelSQLExec.php	(revision 3150)
+++ lib/vendor/propel-generator/classes/propel/phing/PropelSQLExec.php	(working copy)
@@ -667,7 +667,7 @@
     {
         if (!empty($this->tSqlCommand)) {
             $this->parent->log("Executing commands", PROJECT_MSG_INFO);
-            $this->parent->runStatements($tSqlCommand, $out);
+            $this->parent->runStatements($this->tSqlCommand, $out);
         }
 
         if ($this->tSrcFile !== null) {
